infrastructure/zk/src/docker.ts

```diff
@@ -66,8 +70,8 @@ async function dockerCommand(
     }
 }
 
-function defaultTagList(image: string, imageTagSha: string, imageTagShaTS: string) {
-    const tagList = [
+function defaultTagList(image: string, imageTagSha: string, imageTagShaTS: string, protocolVersionTag: string) {
+    let tagList = [
```

Comment:
any reason you change const -> let?

Reply:
To be able to add more tags via push() below.